cognite/client/_api/functions.py

```diff
@@ -766,7 +790,9 @@ def validate_function_folder(root_path: str, function_path: str, skip_folder_val
             _run_import_check_backup(root_path, module_path)
 
 
-def _validate_function_handle(handle_obj: Callable | ast.FunctionDef) -> None:
+def _validate_function_handle(
+    handle_obj: Callable[..., object] | ast.FunctionDef,
```

Comment:
should this be `FunctionHandle | ast.FunctionDef`?

Reply:
No, not here. Verified with @qTipTip